lib/CMakeLists.txt

```diff
@@ -20,7 +20,7 @@ SET_PROPERTY(TARGET  dealii_${TARGET}  APPEND  PROPERTY  COMPILE_DEFINITIONS
             GIT_SHORTREV="${GIT_SHORTREV}")
 
 # the environment variable precice_DIR is searched by default
-FIND_PACKAGE(precice 2.0
+FIND_PACKAGE(precice 2.4
```

Comment:
@renefritze FYI, I would enforce it now this way, all CIs will consequently fail.

Reply:
That's alright with me :+1: